ci/pipeline.yaml

```diff
@@ -100,24 +119,44 @@ jobs:
       - put: version
         inputs: detect
         params: { file: version/version }
-      - in_parallel:
-        - put: pre-release
-          inputs: detect
-          params:
-            name: version/version
-            tag: version/version
-            globs:
-              - build/*
-        - put: docker-image
+      - put: pre-release
+        inputs: detect
+        params:
+          name: version/version
+          tag: version/version
+          globs:
+            - build/*
+      - get: ubuntu-image
+        params:
+          format: oci
+      - file: version/version
+        load_var: image_version
+      - get: oci-build-task
+      - task: build-oci
+        privileged: true
+        image: oci-build-task
+        config:
+          platform: linux
           inputs:
-            - build
-            - source
-            - version
+            - name: source
+            - name: build
+            - name: ubuntu-image
+          outputs:
+            - name: image
           params:
-            build: .
-            dockerfile: source/Dockerfile
-            tag_as_latest: false
-            tag_file: version/version
+            CONTEXT: .
+            DOCKERFILE: source/Dockerfile
+            IMAGE_ARG_ubuntu_image: ubuntu-image/image.tar
+          run:
+            path: build
```

Comment:
Similar to `needs`, not sure if there is a path conflict here or if the prior build task needs a refactor